code/cvar_ref.as

```diff
@@ -0,0 +1,17 @@
+// Demonstrates ConVar refs
+
+[ClientCommand("my_cvarref_demo")]
+void MyCvarRefDemo(const CommandArgs@ args)
+{
+	ConVarRef intensity("r_portal_light_intensity");
+
+	Msg("is r_protal_light_intensity valid? " + intensity.GetBool() + "\n");
+	Msg("current val: " + intensity.GetFloat() + "\n");
+
+	intensity.SetValue(1);
+	Msg("New val: " + intensity.GetFloat() + "\n");
+
+	ConVarRef invalid("my_invalid_thingy");
```

Comment:
I had to double take, I think invalid_cvar or something could be better but I think it's also just fine as it is.

Reply:
Done